spec/support/page_helpers.rb

```diff
@@ -105,4 +105,34 @@ def then_i_see_the(page, **args)
   def when_i_visit_the(page, **args)
     send(page.to_sym).load(**args)
   end
+
+  def teacher_sign_in_page
```

Comment:
I think it would be good if the `then_i_see_the` and `when_i_visit_the` methods are defined either above or below _all_ the page object methods, so it's easier to find them in this file, rather than being in the middle of them.